components/table/Table.razor

```diff
@@ -184,7 +184,7 @@ RenderFragment<(Table<TItem> table, bool header)> colGroup = ctx =>
                 }
                 @if (!hasChildren && table.ExpandTemplate != null && table.RowExpandable(currentRowData))
                 {
-                    <tr class="ant-table-expanded-row ant-table-expanded-row-level-1" style="@(currentRowData.Expanded?"":"display: none;")">
+                    <tr class="ant-table-expanded-row ant-table-expanded-row-level-1 @table.RowClassName(currentRowData)" style="@(currentRowData.Expanded?"":"display: none;")">
```

Comment:
Here is an other peroperty for expand row.

![image](https://user-images.githubusercontent.com/7550366/105566892-7de45f80-5d69-11eb-9faf-cfdcf128fcf0.png)